### PR TITLE
Data share addition

### PR DESCRIPTION
**Pull request by 0xBoxer**
add a page for data shares

### CODE REVIEW

File: data-share.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **Human-Readable Smart Contracts**: Simplifying the complexity of smart contracts, making them accessible and understandable.
+- **Direct Access via Snowflake Marketplace**: Seamless integration and access through the Snowflake data marketplace, ensuring ease of use and efficiency.
+- **Cross Region Replication**: Data is available in multiple regions, ensuring low latency and high availability.
+- **Precision and Accuracy**: Data is meticulously curated and maintained, ensuring the highest quality and reliability.
+
+## Getting Started
+
+To start leveraging Dune's Datashare on Snowflake for your blockchain analytics needs, visit the Dune profile on the [Snowflake Marketplace](https://app.snowflake.com/marketplace/providers/GZSVZMYSCO/Dune). You will find one listing for each blockchain, and a listing for our Cross Chain datasets. For each listing, you will see a only couple of tables available. These are sample datasets for testing and available under free trial. To access the full data catalog and start your subscription, please contact us via the contact form on our [enterprise page](https://dune.com/enterprise).
```
Comment (jonathandune) [inline data-share.mdx:19]: "you will only see a couple"

File: data-share.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Unlock the full potential of blockchain analytics with Dune's Datashare on Snowflake. Gain access to Dune's full data catalog with over 1.5 million tables, featuring raw blockchain data, human-readable smart contracts, detailed insights into protocols, dex/nft trades, and comprehensive cross-chain data. All directly available on the Snowflake Marketplace, making it the most extensive and rich resource for blockchain data in the crypto universe.
+
+## Key Features
+
+- **Extensive Data Catalog**: The largest assembly of blockchain data in the crypto space, at your fingertips.
+- **Cross-Chain Insights**: Abstracted and harmonized data from more than 15 blockchains, providing a panoramic view of the blockchain ecosystem.
+- **Human-Readable Smart Contracts**: Simplifying the complexity of smart contracts, making them accessible and understandable.
+- **Direct Access via Snowflake Marketplace**: Seamless integration and access through the Snowflake data marketplace, ensuring ease of use and efficiency.
+- **Cross Region Replication**: Data is available in multiple regions, ensuring low latency and high availability.
```
Comment (jonathandune) [inline data-share.mdx:14]: would suggest "ensuring optimal performance and reduced query costs"

File: data-share.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+To start leveraging Dune's Datashare on Snowflake for your blockchain analytics needs, visit the Dune profile on the [Snowflake Marketplace](https://app.snowflake.com/marketplace/providers/GZSVZMYSCO/Dune). You will find one listing for each blockchain, and a listing for our Cross Chain datasets. For each listing, you will see a only couple of tables available. These are sample datasets for testing and available under free trial. To access the full data catalog and start your subscription, please contact us via the contact form on our [enterprise page](https://dune.com/enterprise).
+
+### Accessing the Data
+
+Once you have subscribed to Dune's Datashare on Snowflake, you can access the data directly from your Snowflake account. The data is organized into schemas, with each schema representing a blockchain or a cross-chain dataset. You can query the data using SQL, and use the results to power your analytics and insights. 
+
+### Data updates and availability
+
+Currently the data is updated every 24 hours by default, for special cases we can disucss more frequent updates. We replicate the data to multiple regions to ensure low latency and high availability.
```
Comment (jonathandune) [inline data-share.mdx:27]: I suggest "to ensure data proximity and help keep your query costs low"

File: data-share.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+
+```sql
+--Using hash:
+select * from ethereum.transactions where hash = X'efb2e2c26974f72d9f3f04c693db73ecc679dd60'
+--Using hash_hex:
+select * from ethereum.transactions where hash_hex = LOWER('0xEfB2E2c26974F72D9f3f04C693DB73eCc679dd60') -- Not recommended
+```
+
+<Info> Varbinary data in snowflake is displayed as `ABCDEF` but needs to be queried via `X'abcdef'`. There is a delta between the representation and the actual data. </Info>
```
Comment (jonathandune) [inline data-share.mdx:52]: difference instead of delta ?

File: data-share.mdx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+The `VARBINARY` datatype columns are useful when exact precision is needed, but calculations are not required. For example, if you want to join on `token_id` for erc1155 transfers, you can use the `VARBINARY` column `token_id_binary`. Using double for this would not work, as the precision is not sufficient. 
+
+|Database    | Column Name                 | Data Type    | Data Display Sample                                       |
+|------------|--------------------------|-----------|---------------------------------------------------|
+|Dune        | column_name              | INT256    | 123456789012345678901234567890123456789           |
+|Snowflake   | column_name              | DOUBLE    | 3.4567890123456788e+16                            |
+|Snowflake   | column_name_binary       | VARBINARY | X'5ce0e9a56015fec5aadfa328ae398115'               |
+|Snowflake   | column_name_raw          | VARBINARY | X'5ce0e9a56015fec5aadfa328ae398115'               |
+
+<Info> Varbinary data in snowflake is displayed as `ABCDEF` but needs to be queried via `X'abcdef'`. There is a delta between the representation and the actual data. </Info>
```
Comment (jonathandune) [inline data-share.mdx:68]: duplicate?

Comment (0xBoxer) [reply data-share.mdx:68]: intentional duplicate as it is revelant for both data types, if you only read one you might get confused

**Pull request merged**